Benchmarking/Pt_stub/RnD1system_benchmarking_analysis.py

```diff
@@ -0,0 +1,258 @@
+        cvs_ec_only = cvs_ec_only.as_cv()
+        axes_c = cvs_ec_only[3].plot_vs_potential()
+        cvs_ec_vs_pot = axes_c.get_figure()
+        if SAVE_FIGURES is True:
+            cvs_ec_vs_pot.savefig(
+                "./" + EXP_NAME + "CV_vs_potential_EC_biologic" + FIGURE_TYPE)
+
+    elif WHICH_PART == "plot+fit_HER":  # plot and fit the HER QC
+        her = full_data.cut(tspan=[1500, 4000])
+        her.tstamp += 1589
+        axes_her = her.plot_measurement(tspan=[0, 3000])
+        fig_her = axes_her[0].get_figure()
+        fig_her.tight_layout()
+        if SAVE_FIGURES is True:
+            fig_her.savefig("./" + EXP_NAME + "_HER_CP.png")
+        signal = "M2"
+        t_list_clean = find_decay_edge(her, signal)  # if automatically generated
+        # list doesn't catch the right times, cutoff value can be selected manually:
+        # gradient_cutoff=-1E-11 (this value works for HER for the tuning used
+        # in this example dataset.)
+        t_half_h2_list = []
+        for time in t_list_clean:
+            t_half_h2, fig = exp_fit_signal(her, signal=signal, tspan=[time, time+5])
+            if SAVE_FIGURES is True:
+                fig[0].savefig("./" + EXP_NAME + "_" + signal +
+                               f"decay_at_{time:.0f}s." + FIGURE_TYPE)
+            t_half_h2_list.append(t_half_h2)
+        np.savetxt("./" + EXP_NAME + "_" + signal + "_decay_times.csv", t_half_h2_list,
+                   delimiter=", ", fmt='%s')
+
+    elif WHICH_PART == "plot+fit_gas_exchange":  # plot and fit the gas exchange QC
+        gas_exchange = full_data.cut(tspan=[5250, 6200])
+        gas_exchange.tstamp += 5250
+        axes_gas_ex = gas_exchange.ms_plotter.plot_measurement()
+        fig_gas_ex = axes_gas_ex.get_figure()
+        if SAVE_FIGURES is True:
+            fig_gas_ex.savefig("./" + EXP_NAME + "_gas_exchange" + FIGURE_TYPE)
+        times_ar = find_decay_edge(gas_exchange, "M40")  # , gradient_cutoff=-1E-10)
+        times_he = find_decay_edge(gas_exchange, "M4")  # , gradient_cutoff=-1E-10)
+        t_half_list = []
+        for time in times_ar:
+            t_half_ar, fig = exp_fit_signal(gas_exchange, signal="M40", tspan=[time, time+5])
+            if SAVE_FIGURES is True:
+                fig[0].savefig("./" + EXP_NAME + f"_M40_decay_at_{time:.0f}s" + FIGURE_TYPE)
+            t_half_list.append(("M40", t_half_ar))
+        for time in times_he:
+            t_half_he, fig = exp_fit_signal(gas_exchange, signal="M4", tspan=[time, time+5])
+            if SAVE_FIGURES is True:
+                fig[0].savefig("./" + EXP_NAME + f"_M4_decay_at_{time:.0f}s" + FIGURE_TYPE)
+            t_half_list.append(("M4", t_half_he))
+        np.savetxt("./" + EXP_NAME + "_gas_exchange_decay_times.csv", t_half_list,
+                   delimiter=", ", fmt='%s')
+
+    else:
+        raise NameError("WHICH_PART not recognized.")
+
```

Comment:
Here, the same comment as for the other script, about possibly the option to show figures